### PR TITLE
renamed db to hdb

### PR DESCRIPTION
**Pull request by pabowers**
Would like to know if this is what you meant with renaming to hdb

### CODE REVIEW

File: deploy/vm/ansible/roles/cockpit-install/tasks/main.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@
 - name: Log into XSA
   shell: |
     . ~/.bashrc
-    xs login --skip-ssl-validation -a https://{{ ansible_fqdn }}:3{{ sap_instancenum }}30/ -s SAP -u XSA_ADMIN  -p {{ pwd_db_xsaadmin }}
+    xs login --skip-ssl-validation -a https://{{ ansible_fqdn }}:3{{ sap_instancenum }}30/ -s SAP -u XSA_ADMIN  -p {{ pwd_hdb_xsaadmin }}
```
Comment (tniek) [inline deploy/vm/ansible/roles/cockpit-install/tasks/main.yml]: As discussed - this should stay pwd_db_xsaadmin, as 'db' here refers to database as a general concept (not to HDB in particular)

File: deploy/vm/ansible/roles/configure-hsr-hana2/tasks/main.yml [hunk 2]
```diff
@@ -1,20 +1,20 @@
 - name: Backup the database
   shell: |
     . ~/.bashrc
-    hdbsql -d SYSTEMDB -u SYSTEM -p "{{ pwd_db_system }}" -i {{ sap_instancenum }} "BACKUP DATA USING FILE ('initialbackupSYS')"
-    hdbsql -d {{ sap_sid|upper }} -u SYSTEM -p "{{ pwd_db_system }}" -i {{ sap_instancenum }} "BACKUP DATA USING FILE ('initialbackup{{ sap_sid|upper }}')"
+    hdbsql -d SYSTEMDB -u SYSTEM -p "{{ pwd_hdb_system }}" -i {{ sap_instancenum }} "BACKUP DATA USING FILE ('initialbackupSYS')"
```
Comment (tniek) [inline deploy/vm/ansible/roles/configure-hsr-hana2/tasks/main.yml]: See above; this should stay pwd_db_system

File: deploy/vm/ansible/roles/configure-hsr-hana2/tasks/main.yml [hunk 3]
```diff
@@ -1,20 +1,20 @@
...
+    hdbsql -d {{ sap_sid|upper }} -u SYSTEM -p "{{ pwd_hdb_system }}" -i {{ sap_instancenum }} "BACKUP DATA USING FILE ('initialbackup{{ sap_sid|upper }}')"
```
Comment (tniek) [inline deploy/vm/ansible/roles/configure-hsr-hana2/tasks/main.yml]: See above; this should stay pwd_db_system

File: deploy/vm/ansible/roles/datagen/tasks/main.yml [hunk 4]
```diff
@@ -7,6 +7,6 @@
 - name: Execute data generation SQL
   shell: |
     . ~/.bashrc
-    hdbsql -d SYSTEMDB -u SYSTEM -p {{ pwd_db_system }} -i {{ sap_instancenum }} -I /usr/sap/{{ sap_sid|upper }}/HDB{{ sap_instancenum }}/crossjoin.sql
+    hdbsql -d SYSTEMDB -u SYSTEM -p {{ pwd_hdb_system }} -i {{ sap_instancenum }} -I /usr/sap/{{ sap_sid|upper }}/HDB{{ sap_instancenum }}/crossjoin.sql
```
Comment (tniek) [inline deploy/vm/ansible/roles/datagen/tasks/main.yml]: See above; this should stay pwd_db_system

File: deploy/vm/ansible/roles/saphana-install/templates/hdbserver_passwords.j2 [hunk 5]
```diff
@@ -4,7 +4,7 @@
     <root_password><![CDATA[***]]></root_password>
     <sapadm_password><![CDATA[{{ pwd_os_sapadm }}]]></sapadm_password>
     <password><![CDATA[{{ pwd_os_sidadm }}]]></password>
-    <system_user_password><![CDATA[{{ pwd_db_system }}]]></system_user_password>
+    <system_user_password><![CDATA[{{ pwd_hdb_system }}]]></system_user_password>
```
Comment (tniek) [inline deploy/vm/ansible/roles/saphana-install/templates/hdbserver_passwords.j2]: See above; this should stay pwd_db_system

File: deploy/vm/ansible/roles/shine-install/tasks/main.yml [hunk 6]
```diff
@@ -8,7 +8,7 @@
 - name: Log into XSA
   shell: |
     . ~/.bashrc
-    xs login --skip-ssl-validation -a https://{{ ansible_fqdn }}:3{{ sap_instancenum }}30/ -s SAP -u XSA_ADMIN  -p {{ pwd_db_xsaadmin }}
+    xs login --skip-ssl-validation -a https://{{ ansible_fqdn }}:3{{ sap_instancenum }}30/ -s SAP -u XSA_ADMIN  -p {{ pwd_hdb_xsaadmin }}
```
Comment (tniek) [inline deploy/vm/ansible/roles/shine-install/tasks/main.yml]: See above; this should stay pwd_db_system

File: deploy/vm/ansible/roles/shine-install/tasks/main.yml [hunk 7]
```diff
@@ -29,7 +29,7 @@
 - name: Execute SQL to prepare SHINE
   shell: |
     . ~/.bashrc
-    hdbsql -d SYSTEMDB -u SYSTEM -p {{ pwd_db_system }} -i {{ sap_instancenum }} -I /usr/sap/{{ sap_sid|upper }}/HDB{{ sap_instancenum }}/prepare_shine.sql
+    hdbsql -d SYSTEMDB -u SYSTEM -p {{ pwd_hdb_system }} -i {{ sap_instancenum }} -I /usr/sap/{{ sap_sid|upper }}/HDB{{ sap_instancenum }}/prepare_shine.sql
```
Comment (tniek) [inline deploy/vm/ansible/roles/shine-install/tasks/main.yml]: See above; this should stay pwd_db_system

File: deploy/vm/ansible/roles/shine-install/templates/prepare_shine.sql.j2 [hunk 8]
```diff
@@ -1,18 +1,18 @@
 set schema SYS_XS_UAA;
 
-alter user SYS_XS_UAA password "{{ pwd_db_xsaadmin }}";
+alter user SYS_XS_UAA password "{{ pwd_hdb_xsaadmin }}";
```
Comment (tniek) [inline deploy/vm/ansible/roles/shine-install/templates/prepare_shine.sql.j2]: See above; this should stay pwd_db_xsaadmin

File: deploy/vm/ansible/roles/shine-install/templates/prepare_shine.sql.j2 [hunk 9]
```diff
@@ -1,18 +1,18 @@
...
 
-connect SYS_XS_UAA password "{{ pwd_db_xsaadmin }}";
+connect SYS_XS_UAA password "{{ pwd_hdb_xsaadmin }}";
```
Comment (tniek) [inline deploy/vm/ansible/roles/shine-install/templates/prepare_shine.sql.j2]: See above; this should stay pwd_db_xsaadmin

File: deploy/vm/ansible/roles/shine-install/templates/prepare_shine.sql.j2 [hunk 10]
```diff
@@ -1,18 +1,18 @@
...
 
 insert into xsrolecollections(last_modified,description,identity_zone_id,name) values (now(),'''','uaa','SHINE_ADMIN');
 
 insert into xsrolecollection2role(last_modified,rolecollection_identity_zone_id,rolecollection_name,role_name,role_identity_zone_id,role_roletemplate_name,role_roletemplate_app_appid) values (now(),'uaa','SHINE_ADMIN','shine_admin','uaa','shine_admin','shine!i1');
 
-connect SYSTEM password "{{ pwd_db_system }}";
+connect SYSTEM password "{{ pwd_hdb_system }}";
```
Comment (tniek) [inline deploy/vm/ansible/roles/shine-install/templates/prepare_shine.sql.j2]: See above; this should stay pwd_db_system

File: deploy/vm/ansible/roles/shine-install/templates/prepare_shine.sql.j2 [hunk 11]
```diff
@@ -1,18 +1,18 @@
...
 
 DROP USER SHINE_USER;
 
-CREATE RESTRICTED USER SHINE_USER PASSWORD "{{ pwd_db_shine }}" SET PARAMETER EMAIL ADDRESS = '{{ email_shine }}', XS_GIVEN_NAME = '' , XS_FAMILY_NAME = '' , XS_RC_XS_USER_PUBLIC = 'XS_USER_PUBLIC';
+CREATE RESTRICTED USER SHINE_USER PASSWORD "{{ pwd_hdb_shine }}" SET PARAMETER EMAIL ADDRESS = '{{ email_shine }}', XS_GIVEN_NAME = '' , XS_FAMILY_NAME = '' , XS_RC_XS_USER_PUBLIC = 'XS_USER_PUBLIC';
```
Comment (tniek) [inline deploy/vm/ansible/roles/shine-install/templates/prepare_shine.sql.j2]: See above; this should stay pwd_db_shine

File: deploy/vm/ansible/roles/xsa-install/defaults/main.yml [hunk 12]
```diff
@@ -18,10 +18,10 @@ url_xs_services:
 pwd_os_sidadm:
 
 # Password for SYSTEM user (SYSTEMDB)
-pwd_db_system:
+pwd_hdb_system:
```
Comment (tniek) [inline deploy/vm/ansible/roles/xsa-install/defaults/main.yml]: See above; this should stay pwd_db_system

File: deploy/vm/ansible/roles/xsa-install/defaults/main.yml [hunk 13]
```diff
@@ -18,10 +18,10 @@ url_xs_services:
...
 
 # Password for XSAADMIN user
-pwd_db_xsaadmin:
+pwd_hdb_xsaadmin:
```
Comment (tniek) [inline deploy/vm/ansible/roles/xsa-install/defaults/main.yml]: See above; this should stay pwd_db_xsaadmin

File: deploy/vm/ansible/roles/xsa-install/defaults/main.yml [hunk 14]
```diff
@@ -18,10 +18,10 @@ url_xs_services:
...
 
 # Password for SYSTEM user (tenant DB)
-pwd_db_tenant:
+pwd_hdb_tenant:
```
Comment (tniek) [inline deploy/vm/ansible/roles/xsa-install/defaults/main.yml]: See above; this should stay pwd_db_tenant

File: deploy/vm/ansible/roles/xsa-install/templates/hdblcm_xsa.passwords.j2 [hunk 15]
```diff
@@ -6,9 +6,9 @@
     <master_password><![CDATA[***]]></master_password>
     <sapadm_password><![CDATA[***]]></sapadm_password>
     <password><![CDATA[{{ pwd_os_sidadm }}]]></password>
-    <system_user_password><![CDATA[{{ pwd_db_system }}]]></system_user_password>
+    <system_user_password><![CDATA[{{ pwd_hdb_system }}]]></system_user_password>
```
Comment (tniek) [inline deploy/vm/ansible/roles/xsa-install/templates/hdblcm_xsa.passwords.j2]: See above; this should stay pwd_db_system

File: deploy/vm/ansible/roles/xsa-install/templates/hdblcm_xsa.passwords.j2 [hunk 16]
```diff
@@ -6,9 +6,9 @@
...
     <streaming_cluster_manager_password><![CDATA[***]]></streaming_cluster_manager_password>
     <ase_user_password><![CDATA[***]]></ase_user_password>
-    <org_manager_password><![CDATA[{{ pwd_db_xsaadmin }}]]></org_manager_password>
-    <tenantdb_user_password><![CDATA[{{ pwd_db_tenant }}]]></tenantdb_user_password>
+    <org_manager_password><![CDATA[{{ pwd_hdb_xsaadmin }}]]></org_manager_password>
+    <tenantdb_user_password><![CDATA[{{ pwd_hdb_tenant }}]]></tenantdb_user_password>
```
Comment (tniek) [inline deploy/vm/ansible/roles/xsa-install/templates/hdblcm_xsa.passwords.j2]: See above; this should stay pwd_db_tenant

File: deploy/vm/modules/ha_pair/main.tf [hunk 17]
```diff
@@ -173,24 +173,24 @@ module "configure_vm" {
...
-  private_ip_address_db0         = "${var.private_ip_address_db0}"
-  private_ip_address_db1         = "${var.private_ip_address_db1}"
+  private_ip_address_hdb0        = "${var.private_ip_address_hdb0}"
+  private_ip_address_hdb1        = "${var.private_ip_address_hdb1}"
   private_ip_address_lb_frontend = "${var.private_ip_address_lb_frontend}"
   pw_hacluster                   = "${var.pw_hacluster}"
   pw_os_sapadm                   = "${var.pw_os_sapadm}"
   pw_os_sidadm                   = "${var.pw_os_sidadm}"
-  pw_db_system                   = "${var.pw_db_system}"
+  pw_hdb_system                  = "${var.pw_hdb_system}"
```
Comment (tniek) [inline deploy/vm/modules/ha_pair/main.tf]: See above; this should stay pwd_db_system

File: deploy/vm/modules/ha_pair/main.tf [hunk 18]
```diff
@@ -173,24 +173,24 @@ module "configure_vm" {
...
   url_xsa_runtime                = "${var.url_xsa_runtime}"
   url_di_core                    = "${var.url_di_core}"
   url_sapui5                     = "${var.url_sapui5}"
   url_portal_services            = "${var.url_portal_services}"
   url_xs_services                = "${var.url_xs_services}"
   url_shine_xsa                  = "${var.url_shine_xsa}"
-  pwd_db_xsaadmin                = "${var.pwd_db_xsaadmin}"
-  pwd_db_tenant                  = "${var.pwd_db_tenant}"
-  pwd_db_shine                   = "${var.pwd_db_shine}"
+  pwd_hdb_xsaadmin               = "${var.pwd_hdb_xsaadmin}"
```
Comment (tniek) [inline deploy/vm/modules/ha_pair/main.tf]: See above; this should stay pwd_db_xsaadmin

File: deploy/vm/modules/ha_pair/main.tf [hunk 19]
```diff
@@ -173,24 +173,24 @@ module "configure_vm" {
...
+  pwd_hdb_tenant                 = "${var.pwd_hdb_tenant}"
```
Comment (tniek) [inline deploy/vm/modules/ha_pair/main.tf]: See above; this should stay pwd_db_tenant

File: deploy/vm/modules/ha_pair/main.tf [hunk 20]
```diff
@@ -173,24 +173,24 @@ module "configure_vm" {
...
+  pwd_hdb_shine                  = "${var.pwd_hdb_shine}"
```
Comment (tniek) [inline deploy/vm/modules/ha_pair/main.tf]: See above; this should stay pwd_db_shine

File: deploy/vm/modules/ha_pair/terraform.tfvars.template [hunk 21]
```diff
@@ -46,7 +46,7 @@ pw_os_sidadm = "XXX"
 
 # Password for the DB SYSTEM user
 # (In MDC installations, this will be for SYSTEMDB tenant only)
-pw_db_system = "XXX"
+pw_hdb_system = "XXX"
```
Comment (tniek) [inline deploy/vm/modules/ha_pair/terraform.tfvars.template]: See above; this should stay pwd_db_system

File: deploy/vm/modules/ha_pair/variables.tf [hunk 22]
```diff
@@ -70,7 +70,7 @@ variable "pw_bastion_windows" {
   default     = ""
 }
 
-variable "pw_db_system" {
+variable "pw_hdb_system" {
```
Comment (tniek) [inline deploy/vm/modules/ha_pair/variables.tf]: See above; this should stay pwd_db_system

File: deploy/vm/modules/ha_pair/variables.tf [hunk 23]
```diff
@@ -87,17 +87,17 @@ variable "pw_os_sidadm" {
   description = "Password for this specific sidadm, which is an OS user"
 }
 
-variable "pwd_db_shine" {
+variable "pwd_hdb_shine" {
```
Comment (tniek) [inline deploy/vm/modules/ha_pair/variables.tf]: See above; this should stay pwd_db_shine

File: deploy/vm/modules/ha_pair/variables.tf [hunk 24]
```diff
@@ -87,17 +87,17 @@ variable "pw_os_sidadm" {
...
   description = "Password for SHINE user"
   default     = ""
 }
 
-variable "pwd_db_tenant" {
+variable "pwd_hdb_tenant" {
```
Comment (tniek) [inline deploy/vm/modules/ha_pair/variables.tf]: See above; this should stay pwd_db_tenant

File: deploy/vm/modules/ha_pair/variables.tf [hunk 25]
```diff
@@ -87,17 +87,17 @@ variable "pw_os_sidadm" {
...
   description = "Password for SYSTEM user (tenant DB)"
   default     = ""
 }
 
-variable "pwd_db_xsaadmin" {
+variable "pwd_hdb_xsaadmin" {
```
Comment (tniek) [inline deploy/vm/modules/ha_pair/variables.tf]: See above; this should stay pwd_db_xsaadmin

File: deploy/vm/modules/playbook-execution/main.tf [hunk 26]
```diff
@@ -13,12 +13,12 @@ resource null_resource "mount-disks-and-configure-hana" {
      \"sap_instancenum\": \"${var.sap_instancenum}\", \
      \"pwd_os_sapadm\": \"${var.pw_os_sapadm}\", \
      \"pwd_os_sidadm\": \"${var.pw_os_sidadm}\", \
-     \"pwd_db_system\": \"${var.pw_db_system}\", \
+     \"pwd_hdb_system\": \"${var.pw_hdb_system}\", \
```
Comment (tniek) [inline deploy/vm/modules/playbook-execution/main.tf]: See above; this should stay pwd_db_system

File: deploy/vm/modules/playbook-execution/main.tf [hunk 27]
```diff
@@ -29,11 +29,11 @@ resource null_resource "mount-disks-and-configure-hana" {
      \"url_shine_xsa\": \"${var.url_shine_xsa}\", \
      \"url_sapcar_windows\": \"${var.url_sapcar_windows}\", \
      \"url_hana_studio_windows\": \"${var.url_hana_studio_windows}\", \
-     \"pwd_db_xsaadmin\": \"${var.pwd_db_xsaadmin}\", \
+     \"pwd_hdb_xsaadmin\": \"${var.pwd_hdb_xsaadmin}\", \
```
Comment (tniek) [inline deploy/vm/modules/playbook-execution/main.tf]: See above; this should stay pwd_db_xsaadmin

File: deploy/vm/modules/playbook-execution/main.tf [hunk 28]
```diff
@@ -29,11 +29,11 @@ resource null_resource "mount-disks-and-configure-hana" {
...
      \"pw_bastion_windows\": \"${var.pw_bastion_windows}\", \
      \"bastion_username_windows\": \"${var.bastion_username_windows}\", \
-     \"pwd_db_tenant\": \"${var.pwd_db_tenant}\", \
-     \"pwd_db_shine\": \"${var.pwd_db_shine}\", \
+     \"pwd_hdb_tenant\": \"${var.pwd_hdb_tenant}\", \
+     \"pwd_hdb_shine\": \"${var.pwd_hdb_shine}\", \
```
Comment (tniek) [inline deploy/vm/modules/playbook-execution/main.tf]: See above; this should stay pwd_db_shine

File: deploy/vm/modules/playbook-execution/main.tf [hunk 29]
```diff
@@ -29,11 +29,11 @@ resource null_resource "mount-disks-and-configure-hana" {
      \"url_shine_xsa\": \"${var.url_shine_xsa}\", \
      \"url_sapcar_windows\": \"${var.url_sapcar_windows}\", \
      \"url_hana_studio_windows\": \"${var.url_hana_studio_windows}\", \
-     \"pwd_db_xsaadmin\": \"${var.pwd_db_xsaadmin}\", \
+     \"pwd_hdb_xsaadmin\": \"${var.pwd_hdb_xsaadmin}\", \
      \"pw_bastion_windows\": \"${var.pw_bastion_windows}\", \
      \"bastion_username_windows\": \"${var.bastion_username_windows}\", \
-     \"pwd_db_tenant\": \"${var.pwd_db_tenant}\", \
-     \"pwd_db_shine\": \"${var.pwd_db_shine}\", \
+     \"pwd_hdb_tenant\": \"${var.pwd_hdb_tenant}\", \
```
Comment (tniek) [inline deploy/vm/modules/playbook-execution/main.tf]: See above; this should stay pwd_db_tenant

File: deploy/vm/modules/playbook-execution/variables.tf [hunk 30]
```diff
@@ -54,7 +54,7 @@ variable "pw_bastion_windows" {
   description = "The password for the bastion host"
 }
 
-variable "pw_db_system" {
+variable "pw_hdb_system" {
```
Comment (tniek) [inline deploy/vm/modules/playbook-execution/variables.tf]: See above; this should stay pwd_db_system

File: deploy/vm/modules/playbook-execution/variables.tf [hunk 31]
```diff
@@ -72,17 +72,17 @@ variable "pw_os_sidadm" {
   description = "Password for this specific sidadm, which is an OS user"
 }
 
-variable "pwd_db_shine" {
+variable "pwd_hdb_shine" {
```
Comment (tniek) [inline deploy/vm/modules/playbook-execution/variables.tf]: See above; this should stay pwd_db_shine

File: deploy/vm/modules/playbook-execution/variables.tf [hunk 32]
```diff
@@ -72,17 +72,17 @@ variable "pw_os_sidadm" {
...
   description = "Password for SHINE user"
   default     = ""
 }
 
-variable "pwd_db_tenant" {
+variable "pwd_hdb_tenant" {
```
Comment (tniek) [inline deploy/vm/modules/playbook-execution/variables.tf]: See above; this should stay pwd_db_tenant

File: deploy/vm/modules/playbook-execution/variables.tf [hunk 33]
```diff
@@ -72,17 +72,17 @@ variable "pw_os_sidadm" {
...
   description = "Password for SYSTEM user (tenant DB)"
   default     = ""
 }
 
-variable "pwd_db_xsaadmin" {
+variable "pwd_hdb_xsaadmin" {
```
Comment (tniek) [inline deploy/vm/modules/playbook-execution/variables.tf]: See above; this should stay pwd_db_xsaadmin

File: deploy/vm/modules/single_node_hana/single-node-hana.tf [hunk 34]
```diff
@@ -53,18 +53,18 @@ module "configure_vm" {
   url_sap_hdbserver          = "${var.url_sap_hdbserver}"
   pw_os_sapadm               = "${var.pw_os_sapadm}"
   pw_os_sidadm               = "${var.pw_os_sidadm}"
-  pw_db_system               = "${var.pw_db_system}"
+  pw_hdb_system              = "${var.pw_hdb_system}"
```
Comment (tniek) [inline deploy/vm/modules/single_node_hana/single-node-hana.tf]: See above; this should stay pwd_db_system

File: deploy/vm/modules/single_node_hana/single-node-hana.tf [hunk 35]
```diff
@@ -53,18 +53,18 @@ module "configure_vm" {
...
   url_xsa_runtime            = "${var.url_xsa_runtime}"
   url_di_core                = "${var.url_di_core}"
   url_sapui5                 = "${var.url_sapui5}"
   url_portal_services        = "${var.url_portal_services}"
   url_xs_services            = "${var.url_xs_services}"
   url_shine_xsa              = "${var.url_shine_xsa}"
-  pwd_db_xsaadmin            = "${var.pwd_db_xsaadmin}"
-  pwd_db_tenant              = "${var.pwd_db_tenant}"
-  pwd_db_shine               = "${var.pwd_db_shine}"
+  pwd_hdb_xsaadmin           = "${var.pwd_hdb_xsaadmin}"
```
Comment (tniek) [inline deploy/vm/modules/single_node_hana/single-node-hana.tf]: See above; this should stay pwd_db_xsaadmin

File: deploy/vm/modules/single_node_hana/single-node-hana.tf [hunk 36]
```diff
@@ -53,18 +53,18 @@ module "configure_vm" {
...
+  pwd_hdb_tenant             = "${var.pwd_hdb_tenant}"
```
Comment (tniek) [inline deploy/vm/modules/single_node_hana/single-node-hana.tf]: See above; this should stay pwd_db_tenant

File: deploy/vm/modules/single_node_hana/single-node-hana.tf [hunk 37]
```diff
@@ -53,18 +53,18 @@ module "configure_vm" {
...
+  pwd_hdb_shine              = "${var.pwd_hdb_shine}"
```
Comment (tniek) [inline deploy/vm/modules/single_node_hana/single-node-hana.tf]: See above; this should stay pwd_db_shine

File: deploy/vm/modules/single_node_hana/terraform.tfvars.template [hunk 38]
```diff
@@ -46,16 +46,16 @@ pw_os_sidadm = "XXX"
 
 # Password for the DB SYSTEM user
 # (In MDC installations, this will be for SYSTEMDB tenant only)
-pw_db_system = "XXX"
+pw_hdb_system = "XXX"
```
Comment (tniek) [inline deploy/vm/modules/single_node_hana/terraform.tfvars.template]: See above; this should stay pwd_db_system

File: deploy/vm/modules/single_node_hana/terraform.tfvars.template [hunk 39]
```diff
@@ -46,16 +46,16 @@ pw_os_sidadm = "XXX"
...
 
 # Password for the DB XSA_ADMIN user
-pwd_db_xsaadmin = "XXX"
+pwd_hdb_xsaadmin = "XXX"
```
Comment (tniek) [inline deploy/vm/modules/single_node_hana/terraform.tfvars.template]: See above; this should stay pwd_db_xsaadmin

File: deploy/vm/modules/single_node_hana/terraform.tfvars.template [hunk 40]
```diff
@@ -46,16 +46,16 @@ pw_os_sidadm = "XXX"
...
 
 # Password for the DB SYSTEM user for the tenant DB (MDC installations only)
-pwd_db_tenant = "XXX"
+pwd_hdb_tenant = "XXX"
```
Comment (tniek) [inline deploy/vm/modules/single_node_hana/terraform.tfvars.template]: See above; this should stay pwd_db_tenant

File: deploy/vm/modules/single_node_hana/terraform.tfvars.template [hunk 41]
```diff
@@ -46,16 +46,16 @@ pw_os_sidadm = "XXX"
...
 
 # Password for the DB SHINE_USER user (SHINE demo content only)
-pwd_db_shine = "XXX"
+pwd_hdb_shine = "XXX"
```
Comment (tniek) [inline deploy/vm/modules/single_node_hana/terraform.tfvars.template]: See above; this should stay pwd_db_shine

File: deploy/vm/modules/single_node_hana/variables.tf [hunk 42]
```diff
@@ -64,7 +64,7 @@ variable "pw_bastion_windows" {
   default     = ""
 }
 
-variable "pw_db_system" {
+variable "pw_hdb_system" {
```
Comment (tniek) [inline deploy/vm/modules/single_node_hana/variables.tf]: See above; this should stay pwd_db_system

File: deploy/vm/modules/single_node_hana/variables.tf [hunk 43]
```diff
@@ -76,17 +76,17 @@ variable "pw_os_sidadm" {
   description = "Password for this specific sidadm, which is an OS user"
 }
 
-variable "pwd_db_shine" {
+variable "pwd_hdb_shine" {
```
Comment (tniek) [inline deploy/vm/modules/single_node_hana/variables.tf]: See above; this should stay pwd_db_shine

File: deploy/vm/modules/single_node_hana/variables.tf [hunk 44]
```diff
@@ -76,17 +76,17 @@ variable "pw_os_sidadm" {
...
   description = "Password for SHINE user"
   default     = ""
 }
 
-variable "pwd_db_tenant" {
+variable "pwd_hdb_tenant" {
```
Comment (tniek) [inline deploy/vm/modules/single_node_hana/variables.tf]: See above; this should stay pwd_db_tenant

File: deploy/vm/modules/single_node_hana/variables.tf [hunk 45]
```diff
@@ -76,17 +76,17 @@ variable "pw_os_sidadm" {
...
   description = "Password for SYSTEM user (tenant DB)"
   default     = ""
 }
 
-variable "pwd_db_xsaadmin" {
+variable "pwd_hdb_xsaadmin" {
```
Comment (tniek) [inline deploy/vm/modules/single_node_hana/variables.tf]: See above; this should stay pwd_db_xsaadmin

**Comment by pabowers** [comment]
@tniek I believe that I have addressed your comments.  Can you please review again?

**Pull request merged**